### PR TITLE
[IMP] account: add a readonly group

### PR DESCRIPTION
**Pull request by william-andre**
Task [2092079](https://www.odoo.com/web#id=2092079&action=333&active_id=967&model=project.task&view_type=form&menu_id=4720)
Accounting firms that want to give access to their customers avoiding
mistakes and risks will love this profile that can't do anything
wrong... Maybe as well as companies auditors..?

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_ma/data/l10n_ma_chart_data.xml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="ISO-8859-15"?>
 <odoo>
-        <!--
```
Comment (qdp-odoo) [inline addons/l10n_ma/data/l10n_ma_chart_data.xml]: I don't get it...

and the first line still says it's latin-9, not UTF8 
`<?xml version="1.0" encoding="ISO-8859-15"?>`

Comment (william-andre) [reply addons/l10n_ma/data/l10n_ma_chart_data.xml]: Yeah my bad, I just changed the encoding without changing the content. I don't even remember why I did this on this PR. I also have another commit on another branch that puts this in csv. Maybe I should remove this from here?

Comment (qdp-odoo) [reply addons/l10n_ma/data/l10n_ma_chart_data.xml]: yes indeed :-)

**Comment by qdp-odoo** [comment]
@robodoo r+

**Comment by qdp-odoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**